build/source/driver/multi_driver.f90

```diff
@@ -209,6 +209,7 @@ program multi_driver
 integer(i4b),parameter           :: ixRestart_im=1001          ! named variable to print a re-start file once per month
 integer(i4b),parameter           :: ixRestart_id=1002          ! named variable to print a re-start file once per day
 integer(i4b),parameter           :: ixRestart_never=1003       ! named variable to print a re-start file never
+integer(i4b),parameter           :: ixRestart_end=1004         ! named variable to print a re-start file at the end of a run
```

Comment:
It would be easier to follow the logic if `ixRestart_end` was moved up one line.